src/components/organisms/MessagesContainer.vue

```diff
@@ -1,9 +1,11 @@
 <template>
   <div class="flex">
-    <div class="w-full md:w-1/3 md:border md:border-r-0 md:border-gray">
-      <inbox />
+    <div class="w-full md:w-1/3 md:border md:border-r-0 md:border-gray md:h-96">
+      <div class="h-full md:overflow-auto">
```

Comment:
If possible, set this div in the Inbox to wrap the inbox messages and not the H1. This will prevent the h1 to scroll. 

If you can't make it, no worries!

Reply:
I made it work, a flex-col lets you adjust heights easier. Also I added a Easter egg to the inbox messages. Let me know if you'd like it removed for the demo